lib/ukrand/getrandom.c

```diff
@@ -0,0 +1,18 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/* Copyright (c) 2023, Unikraft GmbH and The Unikraft Authors.
+ * Licensed under the BSD-3-Clause License (the "License").
+ * You may not use this file except in compliance with the License.
+ */
+
+#include <string.h>
+#include <sys/random.h>
+#include <uk/essentials.h>
+#include <uk/swrand.h>
+#include <uk/syscall.h>
+
+UK_SYSCALL_R_DEFINE(ssize_t, getrandom,
+		    void *, buf, size_t, buflen,
+		    unsigned int, flags)
+{
+	return uk_swrand_fill_buffer(buf, buflen);
```

Comment:
Are you not getting an unused variable warning for `flags`? If so, consider using either `__unused` or `__maybe_unused`.

Reply:
Not sure what its the proper syntax in case of `UK_SYSCALL_R_DEFINE`, could you provide an example?